curated_transformers/models/hf_wrapper.py

```diff
@@ -95,7 +94,40 @@ def convert_for_torch_backward(dY: List[Floats2d]):
     return Y, convert_for_torch_backward
 
 
-def encoder_from_pretrained_hf_model(
+def bert_encoder_from_pretrained_hf_model(
+    model_name: str, model_revision: str = "main"
+) -> BertEncoder:
+    from transformers import AutoModel, AutoTokenizer
```

Comment:
Don't we need a `global transformers` here?

Reply:
We don't need it, right? But I guess it could be a bit more efficient?